doc/howto/migrate_from_lxc.md

```diff
@@ -16,10 +16,12 @@ However, this tool does not migrate any of the LXC container configuration.
 
 ## Get the tool
 
-If you're using the snap, the `lxc-to-lxd` is automatically installed.
-It is available as `lxd.lxc-to-lxd`.
+To get the `lxc-to-lxd` tool you can either download a pre-built binary:
```

Comment:
```suggestion
To get the `lxc-to-lxd` tool, you can download a pre-built binary:
```

Not a fan of either/or with so much text in between. ;) But it should work fine without.